release-notes/24.05/index.html

```diff
@@ -0,0 +1,150 @@
+            </li>
+            <li>
+            The <code>MAP_CHERI_NOSETBOUNDS</code> flag to the mmap system call is now deprecated and a no-op.  When mmap is called with the <code>MAP_FIXED</code> flag and a pointer to an existing reservation, the passed pointer will be returned unmodified on success.
+            </li>
+          </ul>
+          <p>
+          While CheriBSD is, by definition, an experimental research operating system, with the exception of the benchmark ABI, the above features are not yet considered to be mature.
+          </p>
+          <p>
+          As with previous releases, the default CheriBSD kernel on Morello ships with debugging features enabled, which should be disabled by booting a non-debug kernel before any performance benchmarking is performed.
+          </p>
+          <h2>Known Issues</h2>
+          <ul>
+            <li>
+              <p>
+              ZFS support is lightly tested. The on-disk format is believed to be stable, but it is possible that future releases will be unable to read pools created by this release, particularly pools created on pure-capability kernels.
+              </p>
+            </li>
+          </ul>
+          <h2>Using Morello as a Desktop</h2>
+          <p>
+          New installations of the latest CheriBSD release have an option to enable the desktop packages during the install process.
+          </p>
+          <p>
+          If you have already installed the latest version of CheriBSD but did not install the desktop packages you can add them later; log in as root or use sudo to add the following packages to the system:
+          </p>
+<pre>
+<code>pkg64c install cheri-desktop</code> <b>(provides DRM, Wayland and the general GUI system)</b>
+<code>pkg64 install cheri-desktop-hybrid-extras</code> <b>(provides Firefox and other applications)</b>
+</pre>
+          <p>
+          Make sure that your user ID is in the video group; if not, use:
+          </p>
+<pre>
+<code>pw groupmod video -m username</code>
+</pre>
+          <p>
+          Add the following lines to /boot/loader.conf:
+          </p>
+<pre>
+<code>dtb_load="YES"</code>
+<code>dtb_type="dtb"</code>
+<code>dtb_name="/boot/dtb/arm/morello-soc.dtb"</code>
+</pre>
+          <p>
+          Ensure that a USB keyboard and mouse, as well as an HDMI monitor capable of handling full-HD input (all modern monitors), are connected to the desktop.
```

Comment:
Are there requirements on the firmware version that must be installed?  I recall there being some sadness about USB peripherals in earlier versions?

Reply:
I believe the current answer is "the latest version" which is effectively what the getting started guide says.